ipi/engine/forcefields.py

```diff
@@ -383,13 +383,13 @@ def poll(self):
 
         # We have to be thread-safe, as in multi-system mode this might get
         # called by many threads at once.
-        with self._threadlock:
-            for r in self.requests:
-                if r["status"] == "Queued":
-                    r["status"] = "Running"
-                    r["t_dispatched"] = time.time()
-                    self.evaluate(r)
-                    r["result"][0] -= self.offset  # subtract constant offset
+        # with self._threadlock:
```

Comment:
hey Elia, I think we can't safely uncomment this. could you explain your rationale here, so we could find another solution?